src/main/java/com/conveyal/r5/analyst/Grid.java

```diff
@@ -599,7 +657,7 @@ public static Polygon getPixelGeometry (int x, int y, int zoom) {
 
             if (statusListener != null) statusListener.accept(currentCount, total);
 
-            if (currentCount % 10000 == 0) {
+            if (currentCount % 100 == 0) {
```

Comment:
This is going to produce large amounts of log output on large Shapefiles. Do we need anything more that a few log entries indicating that the file is being processed?

Reply:
Oops, forgot to reset this from testing locally.